src/uwtools/cli.py

```diff
@@ -744,6 +737,27 @@ def _add_arg_total(group: Group) -> None:
     )
 
 
+def _add_arg_update_file(group: Group, required: bool = False) -> None:
+    group.add_argument(
+        _switch(STR.updatefile),
+        "-u",
+        help="Path to update file (defaults to stdin)",
+        metavar="PATH",
+        required=required,
+        type=Path,
+    )
+
+
+def _add_arg_update_format(group: Group, choices: List[str], required: bool = False) -> None:
+    group.add_argument(
+        _switch(STR.updatefmt),
+        choices=choices,
+        help="Input format",
```

Comment:
```suggestion
        help="Update format",
```

Should this be update format since we are passing in the `update file` and not in the input file here?

Reply:
Yes, thanks!